soc/ti/k3/am6x/r5/soc.c

```diff
@@ -53,3 +56,34 @@ void soc_early_init_hook(void)
 {
 	k3_unlock_all_ctrl_partitions();
 }
+LOG_MODULE_REGISTER(soc, LOG_LEVEL_DBG);
+
+#if CONFIG_SOC_AM2434_R5F0_0
```

Comment:
Can we have a new config for this feature and just `imply CONFG_` for each core which supports this? This if will grow over the time.